Website: Add case study article template page and three articles (#37153)
fleetdm/fleet: articles/foursquare.md

They also found that maintaining multiple MDM tools pulled time away from supporting users. They wanted to refocus their resources on serving their users’ needs instead of “looking after the tools that managed their devices.”

## The search for a solution

The team at Foursquare began evaluating alternatives. Their main goal was to consolidate their device management into a single platform that supported both macOS and Windows. Any replacement also had to match the controls they already relied on in their existing MDMs.

They needed a platform with a developer-friendly API — something simple, reliable, and easy to work with. A clear API would let the team invest more in automation, which hadn’t been practical while managing two different MDMs.

They also wanted the migration to be quick and seamless. With their past experience, the team knew that MDM changes could be disruptive, and they wanted to minimize any impact on end users.

## Choosing Fleet

> The Foursquare team had met Fleet at a DevOps conference. After hearing positive feedback and conducting their own research they decided to evaluate the platform. Fleet checked all the boxes they were looking for: macOS and Windows support, a developer-friendly API, and a GitOps-first approach.

Reviewer: ```suggestion
The Foursquare team had met Fleet at a DevOps conference. After hearing positive feedback and conducting their own research, they decided to evaluate the platform. Fleet checked all the boxes they were looking for: macOS and Windows support, a developer-friendly API, and a GitOps-first approach.
```

> During the evaluation, the team was particularly impressed with Fleet’s Open Source roots. The transparency gave them confidence early on. Everything from bugs, to roadmaps and source code was available for review.

Reviewer: ```suggestion
During the evaluation, the team was particularly impressed with Fleet’s Open Source roots. The transparency gave them confidence early on. Everything from bugs to roadmaps and source code was available for review.
```

Fleet has also become a key part of their automation workflow. Managing configurations through automation — instead of through a UI — gave the team a level of confidence they didn’t have before.

<div purpose="attribution-quote">

Fleet is the best tool for Foursquare - it helps my team be more creative and effective.

**Mike Meyer**

Senior Manager, Corporate Engineer

</div>

<meta name="category" value="case study">
<meta name="companyLogoFilename" value="foursquare-logo-212x40@2x.png">
<meta name="articleTitle" value="Foursquare cut costs and gained 114% ROI with Fleet">

<meta name="publishedOn" value="2025-12-11">
<meta name="authorGitHubUsername" value="n/a">
<meta name="authorFullName" value="Fleetdm">

<meta name="quoteContent" value="“One of the easiest, quickest, smoothest migrations I’ve ever done.”">
<meta name="quoteAuthorImageFilename" value="mike-meyer-120x120@2x.png">
<meta name="quoteAuthorName" value="Mike Meyer">
<meta name="quoteAuthorJobTitle" value="Senior Manager, Corporate Engineer">

<meta name="companyName" value="Foursquare">

> <meta name="companyInfo" value="Foursquare is the industry's leading geospatial technology platform, designed to help businesses make smarter decisions and create more engaging customer experiences. Powered by deep machine learning and a privacy-forward approach, their technology and solutions are redefining how organizations derive value from location intelligence.">

Reviewer: ```suggestion
<meta name="companyInfo" value="Foursquare is the industry's leading geospatial technology platform, designed to help businesses make smarter decisions and create more engaging customer experiences.">
<meta name="companyInfoLineTwo" value="Powered by deep machine learning and a privacy-forward approach, their technology and solutions are redefining how organizations derive value from location intelligence.">
```

<meta name="summaryChallenge" value="Foursquare wanted to simplify how they managed Macs and Windows devices. Their devices were split across two different MDMs, which created extra work and made automation harder for a small IT team.">

> <meta name="summarySolution" value="Fleet provided Foursquare with a single MDM platform that supports both Mac and Windows. Migrating from their previous MDMs was seamless and easy, with Fleet-provided out-of-the-box tools to add speed and simplicity.">

Reviewer: ```suggestion
<meta name="summarySolution" value="Fleet provided Foursquare with a single MDM platform that supports both Mac and Windows. Migrating from their previous MDMs was seamless and easy, thanks to Fleet-provided out-of-the-box tools that added speed and simplicity.">
```
